cdc-accelerator/feature/permissionGroups.js

```diff
@@ -5,7 +5,7 @@ import { clearDirectoryContents } from '../utils/utils.js'
 import { SRC_DIRECTORY, BUILD_DIRECTORY } from './constants.js'
 import client from '../../cdc-accelerator/sap-cdc-toolkit/gigya/client.js'
 export default class PermissionGroups extends PartnerFeature {
-    static PERMISSIONGROUP_FILE_NAME = 'PermissionGroups.json'
+    static PERMISSIONGROUP_FILE_NAME = `${this.getName()}.json`
```

Comment:
The variable is static, is created before the instance class, so there is no this at this point.
One option is to not make the variable static and initiate it in the class constructor